ambry-frontend/src/main/java/com.github.ambry.frontend/FrontendMetrics.java

```diff
@@ -137,6 +137,16 @@
   public final Histogram blobStorageServiceStartupTimeInMs;
   public final Histogram blobStorageServiceShutdownTimeInMs;
 
+  // Account
+  public final Counter oldPutUnrecognizedAccountCount;
```

Comment:
Have renamed the metrics. Take a look to see if they could be understandable just by reading their names.
```
  public final Counter unRecognizedServiceIdCount;
  public final Counter unRecognizedAccountNameCount;
  public final Counter missingDefaultContainerCount;
  public final Counter unRecognizedContainerNameCount;
  public final Counter getHeadDeleteUnrecognizedAccountCount;
  public final Counter getHeadDeleteUnrecognizedContainerCount;
  public final Meter putWithServiceIdForAccountNameRate;
  public final Meter putWithAccountAndContainerHeaderRate;
```